test/components/event_component_test.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'view_component/test_case'
+
+class EventComponentTest < ViewComponent::TestCase
+  setup do
+    VCR.use_cassette(:import_test_calendar) do
+      @site = create(:site)
+      @context = 'SomeContext'
+      @event = create(:event)
+      @primary_neighbourhood = create(:neighbourhood)
+      @show_neighbourhoods = false
+    end
+  end
+
+  def test_component_renders_event_without_badge
+    with_request_url('/events/11234.json', host: 'mossley.mossley.localhost') do
```

Comment:
this looks a little odd - normally you'd visit `event_path(@event)`? how come we're loading json?